gunicorn/workers/_gaiohttp.py

```diff
@@ -16,6 +22,12 @@ class AiohttpWorker(base.Worker):
 
     def __init__(self, *args, **kw):  # pragma: no cover
         super().__init__(*args, **kw)
+        cfg = self.cfg
+        if cfg.ssl_version:
```

Comment:
You are right about is_ssl. It was only an idea how to enable ssl for worker.
About wrapping directly: if you mean that `_create_server` method is redundant, consider a situation with subclassing `AiohttpWorker` and doing some magic with factory or socket before/after server creation. 
Without `_create_server` it's necessary to copy-paste and change the whole `_run` method (40 LOC) - not a good idea.